### PR TITLE
Wrlock netinfo ptr when cleaning up

### PR DESCRIPTION
**Pull request by adizaimi**
Wrlock netinfo ptr when cleaning up.
print_net_memstat() to get netinfo_ptr lock.
Increase timeout for verify.

### CODE REVIEW

File: net/net.c [hunk 1]
```diff
@@ -6318,18 +6325,15 @@ int net_set_max_bytes(netinfo_type *netinfo_ptr, uint64_t x)
 int net_sanctioned_list_ok(netinfo_type *netinfo_ptr)
```
Comment (akshatsikarwar) [inline net/net.c]: This looks wrong. You need to break on the first bad node found. Otherwise, answer is the state of the last node in sanc list.

Comment (akshatsikarwar) [reply net/net.c]: Oh wait, you made it harder to read when you rolled that into the for-loop ;)

Comment (adizaimi) [reply net/net.c]:
```suggestion
int net_sanctioned_list_ok(netinfo_type *netinfo_ptr)
```

Comment (adizaimi) [reply net/net.c]: Sorry, will move the ok before the ptr check.

**Pull request merged**